packages/hash/frontend/next.config.js

```diff
@@ -31,59 +30,51 @@ process.env.NEXT_PUBLIC_HASH_OPENSEARCH_ENABLED =
  */
```

Comment:
> @todo try using next-compose-plugins when upgrading next to 11 and/or to webpack 5
>    was not building with compose-plugins on next 10 w/ webpack 4.

This todo is present above the selected line. Is this relevant with the below changes? If it is, is it relevant for us to capture in a task?

Reply:
I was going to add this package first, but then decided not to. We got one less plugin, so fewer things to compose. Maybe we can introduce [next-compose-plugins](https://www.npmjs.com/package/next-compose-plugins) when we need to increase the number of plugins next time. The need for this gets smaller as Next supports more and more features out of box.